lib/connection-manager.js

```diff
@@ -54,10 +54,12 @@ const checkHealth = (config) => {
                         log('LogDNA Agent Update Request Received - Auto Updating...');
                         return utils.updateSelf();
```

Comment:
This turns out to be academic because I learned today we're not supporting auto-update in the future, but do you need to restart on an update?

Reply:
Interesting question - I need (!) so, I need to change the logic little bit more...